reports/drm:i915:intel_ddi:c:enum-conversion:0001.txt

```diff
@@ -0,0 +1,114 @@
+Analysis Report 0001
+Warning Type and Explanation: Enum-Conversion , implicit conversion from enumeration type 'enum port' to different enumeration type 'enum intel_dpll_id' [-Wenum-conversion]
```

Comment:
Make two Tags out of this. Rename Warning Type to just Type.
Rename Explanation to Tool Output. (Is Explanation the full output by the compiler?)